README.md

```diff
@@ -19,7 +19,7 @@ Set your own challenges and be the Dungeon Master of your own game. Make your ow
 
 HouseRules applies customisations to many settings, values and toggles used within the Demeo code.
 
-![RoomFinder Screenshot](docs/HouseRules.png)
+![RoomFinder Screenshot](docs/houserules_screenshot.jpg)
```

Comment:
My fault, but text should say 'HouseRules Screenshot' rather than RoomFinder.

Reply:
oops, good catch.  feel free to hijack this branch if you'd like. else, I'll make the change when I get back online in a few hours

Reply:
Already open in my editor... new commit coming in 5... 4... 3... 2...